### DISCUSSION TITLE
📚 Add your testimonials about me

### DISCUSSION THREAD

**Discussion by MrKrishnaAgarwal** [MrKrishnaAgarwal/MrKrishnaAgarwal#2] [General]
<h3>Welcome to my profile! 👋</h3>

<h3>You can add your testimonial below. 
Write below and let me know about it! 👇</h3>

<h3>Thanks a lot for giving your precious time to visit my profile :)</h3>

**Comment by snowlyg** [comment]


**Reply by MrKrishnaAgarwal** [reply]
Thanks a lot @snowlyg for the appreciation
What do you mean by **"can I use it?"**

I already have [Readme Components Repo](https://github.com/MrKrishnaAgarwal/readme-components-github) if you haven't seen it till now.

If you need any further help, feel free to drop a reply and ping me.

Thanks,
Krishna Agarwal

**Reply by MrKrishnaAgarwal** [reply]
Sorry @snowlyg, if you want to use it you have to give me credits for it at the end of the repo, because I have licensed my repo

**Comment by yogaxv** [comment]
wow! good profile!

**Reply by MrKrishnaAgarwal** [reply]
Thanks a lot!
